frontend/src/pages/MapView/index.jsx

```diff
@@ -52,6 +52,8 @@ class MapView extends Component {
   searchHandler = async () => {
     let searchResults;
     try {
+      // search only passes in keyword
+      // Pass in location too
```

Comment:
let's put any work on this in its own PR

Reply:
yessir